src/debugpy/__main__.py

```diff
@@ -43,3 +43,8 @@
     from debugpy.server import cli
 
     cli.main()
+else:
+    # Instead of hard-coding "debugpy = debugpy.server.cli:main" in setup.py, import it
+    # here. If the fully-qualified name of our `main()` function changes, we don't need
+    # to change it in setup.py, as long as we import it here as `cli.main()`.
+    from debugpy.server import cli
```

Comment:
It's okay to hardcode this. From the perspective of an outside client, `debugpy.server` is indeed an implementation detail; but since `setup.py` belongs to the same package, it can depend on it. While it's not a public API entrypoint by itself, we don't anticipate changing the actual function name or location, so another abstraction layer here is likely to be wasted effort.

Reply:
Noted, I will clean this up.